### ISSUE TITLE
Preset responsibleDistrict for import of entries

### ISSUE THREAD

**Issue by Marko-ilmkreis** [hzi-braunschweig/SORMAS-Project#7281]
### Feature Description

Currently the import of entries requires either a user which has responsibleDistrict or the responsibleDistrict must be set within the file.

Currently the most of the healths office uses the Contact Supervisor, Surveillance Supervisor and Clinician for the most of the users. With non of them a district is set.

The idea is to use the district of the current Sormas instance if none is given during the import of the entries,

**Comment by bernardsilenou** [comment]
@Marko-ilmkreis 
- Does sormamas instance has a district? All the districts in the sormas instance belongs to the intance. The district of the entity is determined not by the district of the user but by the district in the importing file.
- Is the person doing the import having challenges to set the district of the entity in the import file?

**Comment by Marko-ilmkreis** [comment]
@bernardsilenou 
To the first question: I hope that on the ITZ are set the districts correct, but I don't know
To the second question: Yes, the average person does not have an good knowledge how to edit data in the csv. The import of the DEA-files, must be easy, even for a DAU

**Comment by bernardsilenou** [comment]
@Marko-ilmkreis 
- Even if the districts of the sormas instance are set by ITZ, there is no rule in sormas that prevents users from adding more districts to the system. So we can only set the country of the sormas instance.
- I think it would be better to train the users to be able to export the files in .csv. They can do the editting in excel and later export as csv.
- Users having the import user role should normally be advanced users. 
- If this is ok with you then you can close the ticket please.

**Comment by Marko-ilmkreis** [comment]
@bernardsilenou we created a few import templates for different situations (eg eps, school classes as contacts, contacts reported via our website). They are designed for users with no experience, but the user has Todo few steps (saving the original file, importing it into Excel, checking the data, saving the csv, importing in Sormas). 

Even with a very detailed instruction sheet, the import is only done by very few users. The other users are not very good with excel and are therefore not importing data. This is a bottle neck. 

The hope with the DEA import was from our side to have only two steps, saving the dea file and import in sormas.

Is it not possible to set the responsible region for the DEA import in evety instance?

**Comment by bernardsilenou** [comment]
@Marko-ilmkreis 
- I ma not sure that I clearly understood this issue, I think we already integrated  sormas to be able to import data from DEA. Maybe some refinement need to be done?
- Some countries uses sormas differently from how we do in Germany, for example, only one instance for the whole country, so there is no one specific responsible region or district for such an instance. If two GE one  day decide to use one server, then we would then  need to  change the import template again. Thus not good to add data to generic import template.
- One possible solution which would need some time to develop is to add an small app to assist users configure import templates. For example: choose an excel file as obtained from DEA -->  specify jurisdiction data (region, district, facility etc) from a drop-down --> export as csv format that is compatible with sormas. This csv file would then be imported in sormas. 

@kwa20 Any suggestion?

**Comment by Marko-ilmkreis** [comment]
@bernardsilenou 
Yes, thats an refinement. The import is still there, but in many health office they don't have users which are  haven a district, because of the most used roles (contact supervisor, surv. supervisor, clinician).

I think also the best solution would be to have a drop down during the import like you described.
